plugins/CoreAdminHome/Menu.php

```diff
@@ -47,9 +49,22 @@ public function configureAdminMenu(MenuAdmin $menu)
     public function configureTopMenu(MenuTop $menu)
     {
         $url = $this->urlForModuleAction('CoreAdminHome', 'home');
-
         $menu->registerMenuIcon('CoreAdminHome_Administration', 'icon-settings');
         $menu->addItem('CoreAdminHome_Administration', null, $url, 980, Piwik::translate('CoreAdminHome_Administration'));
+
+        $changes = \Piwik\Plugins\CoreHome\Controller::getChanges();
+        if (!Piwik::isUserIsAnonymous() && count($changes['changes']) > 0) {
```

Comment:
just to be save could maybe also check additionally if user has some view access?